src/components/PageNavigationButtons/PageNavigationButtons.scss

```diff
@@ -1,7 +1,19 @@
 .page-nav-btns{
   display: flex;
-  justify-content: center;
+  justify-content: space-between;
   a{
     margin: 0 5px;
   }
+  .btn-label{
+    display: inline-block;
+    div{
```

Comment:
<Icon/>s should be using spans now. You may need to rebase and/or run `npm install` inside the container.